scheduler/task_finder_test.go

```diff
@@ -218,6 +218,14 @@ func (s *TaskFinderComparisonSuite) SetupSuite() {
 
 	s.NoError(ref.Insert())
 
+	ref = &model.ProjectRef{
+		Identifier:          "dispatching-disabled",
+		DispatchingDisabled: true,
+		Enabled:             true,
+	}
```

Comment:
I'm confused how the tests in this file work. Does this actually test that tasks aren't able to be dispatched for this project ref, or is it something else?

Reply:
I think the part of this suite that applies to these changes is the [TestFindRunnableHostsIsIdentical](https://github.com/evergreen-ci/evergreen/blob/84bb055e809b5cb659332c1ecf452e55aa6328ee/scheduler/task_finder_test.go#L269) test. It's making sure each of the 4 methods for finding runnable tasks returns the same list of tasks.

Reply:
I added tests to check tasks aren't returned.